feature/selector.py

```diff
@@ -507,6 +511,14 @@ def benchmark(selectors: Dict[str, Union[SelectionMethod.Correlation,
         Whether to drop features with zero variance before running feature selector methods or not.
     verbose: bool, optional (default=False)
         Whether to print progress messages or not.
+    n_jobs: int, optional (default=1)
+        Number of concurrent processes/threads to use in parallelized routines.
+        If set to -1, all CPUs are used.
+        If set to -2, all CPUs but one are used, and so on.
+    backend: str, optional (default=None)
+        A parallelization backend implementation supported in the joblib library.
+        Supported options are “loky” (used by default), “multiprocessing”, and “threading”.
```

Comment:
confused with this line and the line after. So the default is None, but loky is used by default?

Reply:
Right! When backend=None, Parallel selects "loky" by default, but actually in our case require="sharedmem" forces it to select "threading" instead I believe. At least that's what I see from what Parallel prints. Would it be more appropriate to say that instead?

Reply:
Yes, good point. Since we are setting require="sharedmem" we effectively override the backend in any case. Given that I think it actually makes most sense to remove the "backend" argument.

Reply:
Sounds good! Will remove.